### PR TITLE
TNR-1804: Switched backoff to retry for gunicorn compatibility

### PR DESCRIPTION
**Pull request by soconnor71**
TNR-1804
Switched from backoff to retry, for gunicorn compatibility within cms ecosytem

### CODE REVIEW

File: vacasa/connect/connect.py [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
 import hashlib
 import hmac
-from typing import Optional
-from urllib.parse import urlparse, urlunparse
-
-import backoff
 import pendulum
 import requests
+from typing import Optional
+from urllib.parse import urlparse, urlunparse
```
Comment (engstrom) [inline vacasa/connect/connect.py]: You're messing up the feng shui of the imports. The PEP8 guidance on imports says:

> Imports should be grouped in the following order:
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.
>
> You should put a blank line between each group of imports.

https://www.python.org/dev/peps/pep-0008/#imports

Comment (soconnor71) [reply vacasa/connect/connect.py]: I did a reformat code, and it didn't seem to find any issues, but I can fix this

Comment (bijan-vacasa) [reply vacasa/connect/connect.py]: 👍 https://www.jetbrains.com/help/pycharm-edu/optimizing-imports.html

Comment (soconnor71) [reply vacasa/connect/connect.py]: This is what optimize imports gives me:
import hashlib
import hmac
from typing import Optional
from urllib.parse import urlparse, urlunparse

import pendulum
import requests
from retry import retry

**Pull request merged**